examples/worlds/sensors_demo.sdf

```diff
@@ -10,16 +10,16 @@
     </physics>
     <plugin
       filename="ignition-gazebo-physics-system"
-      name="ignition::gazebo::systems::Physics">
+      name="gz::sim::systems::Physics">
```

Comment:
These changes need to be looked at more closely, this is not working:

```
[ignition::plugin::Loader::LookupPlugin] Failed to get info for [gz::sim::systems::Physics]. Could not find a plugin with that name or alias
.                                                                                                                                           
[Err] [SystemLoader.cc:93] Failed to load system plugin [gz::sim::systems::Physics] : could not instantiate from library [ignition-gazebo-ph
ysics-system] from path [/home/chapulina/dev_focal/ws_citadel/install/lib/ign-gazebo-3/plugins/libignition-gazebo-physics-system.so].       
[ignition::plugin::Loader::LookupPlugin] Failed to get info for [gz::sim::systems::Sensors]. Could not find a plugin with that name or alias
.                                                                                                                                           
[Err] [SystemLoader.cc:93] Failed to load system plugin [gz::sim::systems::Sensors] : could not instantiate from library [ignition-gazebo-se
nsors-system] from path [/home/chapulina/dev_focal/ws_citadel/install/lib/ign-gazebo-3/plugins/libignition-gazebo-sensors-system.so].
[ignition::plugin::Loader::LookupPlugin] Failed to get info for [gz::sim::systems::SceneBroadcaster]. Could not find a plugin with that name
 or alias.
[Err] [SystemLoader.cc:93] Failed to load system plugin [gz::sim::systems::SceneBroadcaster] : could not instantiate from library [ignition-
gazebo-scene-broadcaster-system] from path [/home/chapulina/dev_focal/ws_citadel/install/lib/ign-gazebo-3/plugins/libignition-gazebo-scene-b
roadcaster-system.so].
```

Probably need to add aliases for all plugins like we have on Garden.

Reply:
Plugin aliases have been added.